code/API_definitions/blockchain-public-address.yaml

```diff
@@ -58,13 +58,13 @@ info:
   license:
```

Comment:
Please change the link to the image : it currently includes `/main/` in the path, which must be replace with `/r2.2/`